### PR TITLE
Feature: file get endpoint

### PR DESCRIPTION
**Pull request by MaksGovor**
- [x] the style and format of the code corresponds to the style of the organization (yarn lint:format)

### CODE REVIEW

File: src/controller/file_manager.controller.ts [hunk 1]
```diff
@@ -5,9 +5,15 @@ import { ApiBearerAuth, ApiResponse, ApiTags } from '@nestjs/swagger';
...
-import { FolderResponse, FileManagerListResponse, SignedPostUrlsResponse } from '../interface/apiResponse';
+import {
+  FolderResponse,
+  FileManagerListResponse,
+  SignedPostUrlsResponse,
+  SignedGetUrlsResponse,
+} from '../interface/apiResponse';
 import {
   CreateFolderRequest,
+  DownloadFileRequest,
```
Comment (tedi4t) [inline src/controller/file_manager.controller.ts:16]: Can you please change naming from DownloadFileRequest to something related to urls. And the same for UploadFileRequest

File: src/controller/file_manager.controller.ts [hunk 2]
```diff
@@ -64,6 +70,20 @@ export class FileManagerController {
...
     const fileSignedPostUrls = await this.fileManagerService.getSignedPostUrls(user, body);
-    return this.fileManagerFormatter.toLinksResponse(fileSignedPostUrls);
+    return this.fileManagerFormatter.toLinksResponsePost(fileSignedPostUrls);
+  }
+
+  @Post('/signed-urls/get')
+  @HttpCode(HttpStatus.OK)
+  @ApiBearerAuth('authorization')
+  @ApiResponse({ status: HttpStatus.OK, type: SignedGetUrlsResponse })
+  public async getSignedGetUrls(
```
Comment (tedi4t) [inline src/controller/file_manager.controller.ts:80]: Can you please change HTTP method from POST to GET, and the same for getSignedPostUrls. And from POST to DELETE for delete request

File: src/service/file_manager/file_manager.formatter.ts [hunk 3]
```diff
@@ -33,14 +34,24 @@ export class FileManagerFormatter {
     };
   }
 
-  public toLinksResponse (signedPostUrls: Array<FileSignedPostUrl>): SignedPostUrlsResponse {
+  public toLinksResponsePost(signedPostUrls: Array<FileSignedPostUrl>): SignedPostUrlsResponse {
```
Comment (tedi4t) [inline src/service/file_manager/file_manager.formatter.ts:37]:
```suggestion
  public toPostUrlsResponse(signedPostUrls: Array<FileSignedPostUrl>): SignedPostUrlsResponse {
```

File: src/service/file_manager/file_manager.formatter.ts [hunk 4]
```diff
@@ -33,14 +34,24 @@ export class FileManagerFormatter {
...
       key: file.key,
       url: signedPost.url,
       fields: signedPost.fields,
     }));
 
-    return { urls }
+    return { urls };
+  }
+
+  public toLinksResponseGet(signedPostUrls: Array<FileSignedGetUrl>): SignedGetUrlsResponse {
```
Comment (tedi4t) [inline src/service/file_manager/file_manager.formatter.ts:48]:
```suggestion
  public toGetUrlsResponse(signedPostUrls: Array<FileSignedGetUrl>): SignedGetUrlsResponse {
```

File: src/service/storage/storage.service.ts [hunk 5]
```diff
@@ -73,27 +73,23 @@ export class StorageService {
...
+  public async getSignedGetUrl(key: string): Promise<string> {
     const input: GetObjectCommandInput = {
       Bucket: this.s3Config.bucket,
       Key: key,
     };
 
     const command = new GetObjectCommand(input);
 
-    return await getSignedUrl(this.client, command, { expiresIn });
+    return await getSignedUrl(this.client, command, { expiresIn: this.s3Config.linkTtl });
```
Comment (tedi4t) [inline src/service/storage/storage.service.ts:84]: Can it be useful to separate linkTtl for upload and download? What do you think?

Comment (MaksGovor) [reply src/service/storage/storage.service.ts:84]: I agree, it is better to make different variables for this, as it is likely better to give more time to upload

File: src/controller/file_manager.controller.ts (hunk 1 above)
Comment (YAGoOaR) [reply src/controller/file_manager.controller.ts:16]: I'll change them to ReceiveUrlGetRequest & ReceiveUrlPostRequest then.

File: src/controller/file_manager.controller.ts (hunk 2 above)
Comment (YAGoOaR) [reply src/controller/file_manager.controller.ts:80]: Done

File: src/service/file_manager/file_manager.formatter.ts (hunk 3 above)
Comment (YAGoOaR) [reply src/service/file_manager/file_manager.formatter.ts]: Ok

File: src/service/file_manager/file_manager.formatter.ts (hunk 4 above)
Comment (YAGoOaR) [reply src/service/file_manager/file_manager.formatter.ts]: Done

**Review by tedi4t — APPROVED** [review]

**Pull request merged**